src/tests/ftest/daos_test/daos_core_test-faults.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2019 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from daos_core_base import DaosCoreBase
+
+
+class DaosCoreTest(DaosCoreBase):
+    # pylint: disable=too-many-ancestors
+    """Runs the non-rebuild daos_test tests with fault injectors enabled.
+
+    :avocado: recursive
+    """
+
+    def test_faults(self):
+        """Jira ID: DAOS-4693.
+
+        Test Description: Run daos_test with fault injection enabled
+
+        Use Cases: core tests for daos_test with fault injection
+
+        :avocado: tags=all,pr,hw,ib2,medium,daos_test_faults
```

Comment:
Do we really want to *replace* the `daos_test` tag, or just add a new tag to indicate the need for FI, like we do with the `hw` tag.  Notice there we didn't replace the `pr` tag with `pr_hw` but rather just added the `hw` tag as an additional filter.

Reply:
I was not sure what we wanted to do here.   Should I change it to:
tags=all,pr,hw,ib2,medium,daos_test,faults   for daos_core_tests-faults
and 
tags=all,pr,hw,ib2,medium,daos_test for daos_core_tests?

Reply:
Changed the tags to:
all,pr,hw,ib2,medium,daos_test
and
all,pr,hw,ib2,medium,daos_test,faults

The jenkins file can include tag => faults or -faults to run the correct daos_core_test